src/app/modules/shared/components/serial-output/serial-output.component.ts

```diff
@@ -13,6 +13,14 @@ import 'chartjs-adapter-moment';
 import { DialogState } from 'src/app/state/dialog.state';
 import { RobotWiredState } from 'src/app/state/robot.wired.state';
 import {MatDialog, MatDialogRef} from "@angular/material/dialog";
+import * as Papa from 'papaparse';
+import { Subscription } from 'rxjs';
+
+declare global {
```

Comment:
ik denk dat dit niet meer nodig is nu de MS IE zooi er uit is?

Reply:
klopt inderdaad, inmiddels veranderd